src/SAWScript/Crucible/LLVM/Skeleton.hs

```diff
@@ -98,6 +98,13 @@ makeLenses ''ModuleSkeleton
 
 parseType :: LLVM.Type -> IO TypeSkeleton
 parseType (LLVM.PtrTo t) = pure $ TypeSkeleton t True [SizeGuess 1 True "default guess of size 1"]
+-- It is unclear how to combine opaque pointers with type skeletons due to the
+-- lack of a pointee type. For now, we simply fail if we encounter one
+-- (see #1877).
+parseType LLVM.PtrOpaque = fail $ unlines
```

Comment:
Not a huge deal, but it would be nice to use `panic` here; it's already a cabal dependency.

Reply:
Just to confirm, isn't the intention behind `panic` that it is used for code that shouldn't be reachable? (This code can definitely be reached by users.) Or is the idea that we should make this case `panic` so that users are more likely to file a bug if they encounter this issue in the wild?

Reply:
IMO, both scenarios are applicable to `panic`: it should never be reached, or it should flag a location where we really would appreciate user feedback with enough associated information that we can identify the release and location to address the issue.

Reply:
I'm convinced. I've upgraded this failure (as well as a similar failure involving `llvm_boilerplate`) to a `panic`.